liblouis/lou_backTranslateString.c

```diff
@@ -1043,6 +1046,7 @@ makeCorrections(const TranslationTableHeader *table,
 				}
 				tryThis++;
 			}
+		posIncremented = 1;
```

Comment:
I don't quite understand. You are definitely sure that at this point the position has been incremented? Why not compare the actual `pos` to a `posBefore` that you set at the beginning of the `while`. You seem to be doing this below in the `CTO_Correct` case.

Reply:
No I'm not sure of it, it's just the initial value. Yeah we could move the `int posBefore = pos` at the beginning of the loop and the `posIncremented = pos > posBefore` to the end of the loop. That might indeed be easier to read.